02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -84,7 +76,7 @@ Here's what has been added to the above since then:
 | A Guide to the PureScript Numeric Hierarchy | Explanation | New Learners | Read the Docs
 | Make the Leap from JavaScript to PureScript | Getting Started + Explanation | Javascript developers | Blog Post series
 
-## Why is PureScript's Documentation Lacking?
+## What are Factors Contributing to PureScript's Documentation Quality?
```

Comment:
This is my only issue with this PR. I think we should use `affecting` rather than `contributing to`:
> What are the Factors Affecting PureScript's Documentation Quality?

"Contributing to" sounds like these factors are only beneficial rather than simply describing the nuanced reality we find ourselves in

Reply:
Yeah, I agree with that. I made the change, and I'll merge it now.